sdv/metadata/single_table.py

```diff
@@ -300,6 +300,28 @@ def update_column(self, column_name, **kwargs):
         self.columns[column_name] = _kwargs
         self._updated = True
 
+    def update_columns(self, column_names, **kwargs):
+        """Update multiple columns with the same metadata kwargs.
+
+        Args:
+            column_names (list[str]):
+                A list of column names to be updated.
+            **kwargs (type):
+                Any key word arguments that describe metadata for the column.
+        """
+        for column_name in column_names:
+            self.update_column(column_name, **kwargs)
```

Comment:
I wonder if it would be better to do a try/catch here. I see 2 potentially confusing issues:
1) user has provided incompatible kwargs - in this case, the user would only see an error for the first column
2) user has provided kwargs incompatible with some, but not all, columns - the user might end up with some columns updated and some not, depending on where the error happened. They'd also only see the error for the first incompatible column they found.

Reply:
Yes, I addressed this here [085ea4d](https://github.com/sdv-dev/SDV/pull/1824/commits/085ea4dd27040c1d38d42eb603798d2c8695e321). Andrew noticed me that we have a similar situation in RDT and that we update only when everything is valid. I tried to do the same here to be consistent.